tests/BUILD.bazel

```diff
@@ -439,7 +439,7 @@ java_fuzz_target_test(
         "--instrumentation_excludes=com.example.SimpleProto*",
         "--custom_hook_excludes=com.example.SimpleProto*",
         # Limit runs to catch regressions in mutator efficiency and speed up test runs.
-        "-runs=40000",
+        "-runs=50000",
```

Comment:
If this gets worse rather than better, we may have failed to achieve our original objective :-D We could look into this tomorrow.

Reply:
The number of runs could be reduced to the original values again.

As discussed, I added a parameter to configure the frequency of cross overs, which probably helped here, as cross overs are not executed every other time now.